### DISCUSSION TITLE
*.xyz TLD is being blocked widely, maybe it is worth changing it

### DISCUSSION THREAD

**Discussion by djuarezg** [antonym/netboot.xyz#700] [General]
I know that according to https://blog.netboot.xyz/2015/11/25/introducing-netboot-xyz/ the `*.xyz` TLD was chose due to its price, but I just wanted to let you know that this domain is starting to be widely blocked by DNS, spam filters, etc.

i.e. our infra DNS is blocking this TLD without any simple way of allowing this specific network.xyz domain

**Comment by juddgaddie** [comment]
Are you using a public DNS provider, like Cloudflare or Google or is this some internal corporate DNS service that is just blocking all .xyz?

**Reply by djuarezg** [reply]
Internal DNS blocking all .xyz.

But this was more a comment about this TLD being broadly blocked by default in many services.

**Comment by antonym** [comment]
There are many valid domains running the xyz tld, including Alphabet (abc.xyz).  I would suggest working with your internal IT to unblock the domain.

If there are security appliances or other services you know of blocking the domain, please list them here and we can make appeals to unblock them as needed.  Some businesses will block because of some of the security programs or utilities you can install from netboot.xyz.
